Runtimes/Core/Concurrency/CMakeLists.txt

```diff
@@ -125,6 +126,7 @@ target_link_libraries(swift_Concurrency PRIVATE
   swiftConcurrencyInternalShims
   $<$<BOOL:${BUILD_SHARED_LIBS}>:swiftThreading>
   $<$<PLATFORM_ID:Windows>:Synchronization>
+  $<$<PLATFORM_ID:Windows>:WinSDK>
```

Comment:
CC: @etcwilde - I'm not sure how this works. This is an upwards dependency on the Platform Overlay which is dependent on core. This is a circular dependency now.

Reply:
If the desire is that e.g. `WinSDK` should be able to use `_Concurrency`, then we should split the former into e.g. `WinSDKCore` and `WinSDK`, with `WinSDKCore` usable from here and `WinSDK` importing and re-exporting `WinSDKCore` as well as any Concurrency-related extensions.

Reply:
As for how it works currently, `WinSDK.swift` isn't using `_Concurrency`, and implicit `_Concurrency` import is disabled in the overlays in the new build system.

Reply:
It is the former - we cannot use `WinSDK` from `_Concurrency` because it is part of the "Core" project which is built before the overlays are ready because the overlays are dependent on "Core".

For what you are trying to do - use `WinSDK` from `_Concurrency`, we should be able to accommodate that by moving `_Concurrency` from "Core" to "Supplemental".

Reply:
(FWIW, it is just more stuff that will need to be fixed, similar to the tests)

Reply:
Another possibility is just to create a whole new repository for these and have them as a separate Swift package.